src/BlazorWebView/src/core/Android/BlazorWebViewHandler.Android.cs

```diff
@@ -69,17 +66,14 @@ private void StartWebViewCoreIfPossible()
 				throw new InvalidOperationException($"Can't start {nameof(BlazorWebView)} without native web view instance.");
 			}
 
-			var resourceAssembly = RootComponents?[0]?.ComponentType?.Assembly;
-			if (resourceAssembly == null)
-			{
-				throw new InvalidOperationException($"Can't start {nameof(BlazorWebView)} without a component type assembly.");
-			}
+			var assetConfig = Services!.GetRequiredService<BlazorAssetsAssemblyConfiguration>()!;
 
 			// We assume the host page is always in the root of the content directory, because it's
 			// unclear there's any other use case. We can add more options later if so.
-			var contentRootDir = Path.GetDirectoryName(HostPage) ?? string.Empty;
+			var contentRootDir = Path.GetDirectoryName(HostPage!) ?? string.Empty;
```

Comment:
Are we enabling nullability by default in these projects? I'm surprised to see the `!` here.

Reply:
Oh never mind, this is not the actual template I guess.

Reply:
Most (or all?) of MAUI has nullable enabled, so I'm keeping that alive and well. After the initial huge overhead of adding it, it's quite easy to maintain! But yeah the project templates won't have this.